frontend/src/components/audit-log/AuditEntryVerbose.vue

```diff
@@ -501,6 +501,16 @@
         <span>Nodes have been removed via the "Manage Palette" option inside Node-RED</span>
     </template>
 
+    <template v-else-if="entry.event === 'resource.cpu'">
+        <label>Instance High CPU usage</label>
+        <span>Instance has spent more than 5 minutes at more than 75% of CPU limit</span>
```

Comment:
I'm slightly concerned about having these numbers hardcoded here. If we find we want to tune the values, we're then stuck in how it gets reported to the user.

Should the audit log event include this information and we use it to generate the full message?

Reply:
Sorted, at the moment it defaults to minutes, do we want to use `moment.humanize`?

Reply:
I think until we have real experience of the results of this, it's hard to say. This'll do for now.